src/mbgl/util/chrono.cpp

```diff
@@ -3,6 +3,13 @@
 #include <parsedate/parsedate.h>
 
 #include <cstdio>
+#include <ctime>
+
+#if defined(_WINDOWS)
+#define _gmtime(t, i) gmtime_s(i, t)
+#else
+#define _gmtime(t, i) gmtime_r(t, i)
```

Comment:
Sadly Linux does not implement `gmtime_s` even if you define `__STDC_WANT_LIB_EXT1__`.